corehq/apps/accounting/tasks.py

```diff
@@ -692,7 +692,7 @@ def assign_explicit_community_subscription(domain_name, start_date, method, acco
     )
 
 
-@periodic_task(serializer='pickle', run_every=crontab(minute=0, hour=9), queue='background_queue', acks_late=True)
+@periodic_task(run_every=crontab(minute=0, hour=9), queue='background_queue', acks_late=True)
 def run_downgrade_process(today=None):
```

Comment:
This function is only ever queued by celery beat - hence no arguments are passed.